### PR TITLE
fix(Tabs): wrap each form with FormProvider and use useFormContext in…

### PR DESCRIPTION
**Pull request by ItayBenMoshe**
…stead of moving down props

### CODE REVIEW

File: client/src/commons/LocationInputField/LocationInput.tsx [hunk 1]
```diff
@@ -27,7 +27,6 @@ export interface GeocodeResponse extends google.maps.GeocoderResult {
     description?: string;
 }
 
-const renderOption = LocationOptionItem;
```
Comment (NaamanTsur) [inline client/src/commons/LocationInputField/LocationInput.tsx:30]: Why was this removed?

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/commons/LocationInputField/LocationInput.tsx (hunk 1 above)
Comment (ItayBenMoshe) [reply client/src/commons/LocationInputField/LocationInput.tsx:30]: it's never used

**Review by NaamanTsur — DISMISSED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Review by nevobonneh — DISMISSED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Review by nevobonneh — APPROVED** [review]
